### PR TITLE
Address review concerns from PR 39

### PR DESCRIPTION
**Pull request by dvdplm**
Follow-up to https://github.com/topos-protocol/ice-frost/pull/39

- Updated with latest `main`
- Address review concerns wrt docs

### CODE REVIEW

File: src/sign/signature.rs [hunk 1]
```diff
@@ -453,43 +468,38 @@ impl<C: CipherSuite> SignatureAggregator<C, Initial<'_>> {
...
-    /// Get the list of partipating signers.
-    /// It will internally sort the signers by indices, remove any duplicate,
-    /// and then return a reference to the updated internal list.
-    ///
-    /// # Returns
-    ///
-    /// A `&Vec<Signer>` of the participating signers in this round.
-    pub fn get_signers(&'_ mut self) -> &'_ Vec<Signer<C>> {
+        // Deduplicate partipating [`Signer`]s and check that there is not
+        // more signers than the number of participants as set in the [`ThresholdParameters`].
```
Comment (Nashtare) [inline src/sign/signature.rs:489]:
```suggestion
        // Deduplicate partipating [`Signer`]s and check that there are no
        // more signers than participants as set in the [`ThresholdParameters`].
```

**Review by Nashtare — APPROVED** [review]
Thank you David! Just a tiny nit regarding a sentence, otherwise LGTM!

**Review by Nashtare — APPROVED** [review]
LGTM!

**Comment by dvdplm** [comment]
@Nashtare Not sure what CI is angry about tbh. :/

**Comment by Nashtare** [comment]
Yeah I've no idea either. I ran all jobs locally so I'll merge as is, and advise if it happens again.

**Pull request merged**